readium/shared/src/main/java/org/readium/r2/shared/publication/protection/ProtectionRetriever.kt

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2023 Readium Foundation. All rights reserved.
+ * Use of this source code is governed by the BSD-style license
+ * available in the top-level LICENSE file of the project.
+ */
+
+package org.readium.r2.shared.publication.protection
+
+import org.readium.r2.shared.asset.Asset
+import org.readium.r2.shared.util.mediatype.MediaTypeRetriever
+
+/**
+ * Retrieves [ContentProtection] schemes of assets.
+ */
+class ProtectionRetriever(
```

Comment:
How about `ContentProtectionRetriever` or `ContentProtectionSchemeRetriever`? For consistency with the other `Retriever` and all the things related to the DRM are named `ContentProtection`.